noah/src/main/scala/edu/uci/ics/cloudberry/noah/kafka/ConsumerKafka.scala

```diff
@@ -0,0 +1,94 @@
+package edu.uci.ics.cloudberry.noah.kafka
+
+import java.io.File
+import java.util.Properties
+
+import com.typesafe.config.ConfigFactory
+import edu.uci.ics.cloudberry.noah.feed.{AsterixHttpRequest, Config, TagTweet, TwitterFeedStreamDriver}
+import org.apache.kafka.clients.consumer.{ConsumerRecords, KafkaConsumer}
+import org.kohsuke.args4j.CmdLineException
+import twitter4j.TwitterException
+
+import scala.collection.JavaConversions._
+
+/**
+  * Created by Monique on 7/18/2016.
+  */
+object ConsumerKafka {
```

Comment:
the filename doesn't explain what this file used for. is it a general kafka consumer?